lib/ama_layout/navigation.yml

```diff
@@ -14,6 +14,12 @@ member:
         link: "<%= Rails.configuration.youraccount_site %>/billing"
       - text: "Print/Request New Card"
         link: "<%= Rails.configuration.youraccount_site %>/membership"
+  - text: "My Roadside Assistance"
+    link: "<%= Rails.configuration.automotive_site %>/"
```

Comment:
I guess we'll have to add this to application.rb for each app with the nav, and to the dev, staging and prod envs for each app as well.